src/main/java/com/algorand/algosdk/v2/client/model/ErrorModel.java

```diff
@@ -0,0 +1,74 @@
+package com.algorand.algosdk.v2.client.model;
+
+import java.util.Objects;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+public class ErrorModel {
```

Comment:
This type probably needs to be handled in the REST framework

Reply:
Will address this once the e2e testing is enabled.

Reply:
Fixed.